packages/api-server/setup.py

```diff
@@ -26,7 +26,7 @@
         "tortoise-orm~=0.18.1",
         "pyjwt[crypto]~=2.4",
         "pydantic~=1.9",
-        "schedule~=1.1.0",
+        "schedule~=1.2.1",
```

Comment:
Is the timezone feature not available in 1.1.0?

Reply:
nope, unfortunately, it was in 1.2.0, but 1.2.1 comes with some bug fixes.
https://github.com/dbader/schedule/pull/576
https://github.com/dbader/schedule/pull/600